### ISSUE TITLE
Output a FlarmView compatible file

### ISSUE THREAD

**Issue by clementallen** [Turbo87/united-flarmnet#85]
Thanks for this repository!  It's been a bugbear of mine for a while that the OGN DDB registrations couldn't be loaded into flight devices and this repo solves that perfectly.

Is it possible to also output a FlarmView formatted file (ending `.flx`) alongside the two for XCSoar and LX devices?

There is a FlarmNet Converter Tool that does this (sadly also just a windows only executable) but I've not been able to find much info on how it's formatted etc unfortunately.  This tool can be downloaded here: https://gliding.lxnav.com/lxdownloads/software/

Thanks again!

**Comment by Turbo87** [comment]
hmm, I've never seen a `.flx` file before. Do you have an example file somewhere? It might be possible to disassemble the `fln2flx.exe` file back to C# code, but it looks like that won't be as trivial as I hoped :D

**Comment by clementallen** [comment]
Certainly!  This is one generated from `united-lx.fln` using the FlarmNet Converter Tool.  (Github won't let me upload files with this extension so I've uploaded to Google Drive)

https://drive.google.com/file/d/1Fp2LePgAT9AD8Kb0minr2kuvdsyZFhis/view?usp=sharing

**Comment by Turbo87** [comment]
hmm, on first glance it looks like a simple format of

- six bytes ASCII flarm ID (don't ask me why they use ASCII for the hex address...)
- one NULL byte as delimiter
- competition ID in ASCII filled with NULL bytes until seven bytes length (or maybe also six bytes plus NULL byte as delimiter)
- repeat...

I think that should be doable. I'm a quite busy the next weeks though, so might take a while until I can get around to it.
